packages/wallets/src/features/cashier/components/WalletCashierContent/__tests__/WalletCashierContent.spec.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { createMemoryHistory, To } from 'history';
+import { Router } from 'react-router-dom';
+import { render, screen } from '@testing-library/react';
+import { WalletDeposit } from '../../../flows/WalletDeposit';
+import { WalletFiatOnRamp } from '../../../flows/WalletFiatOnRamp';
+import { WalletResetBalance } from '../../../flows/WalletResetBalance';
+import { WalletTransactions } from '../../../flows/WalletTransactions';
+import { WalletTransfer } from '../../../flows/WalletTransfer';
+import { WalletWithdrawal } from '../../../flows/WalletWithdrawal';
+import WalletCashierContent from '../WalletCashierContent';
+
+jest.mock('../../../flows/WalletDeposit');
+jest.mock('../../../flows/WalletFiatOnRamp');
+jest.mock('../../../flows/WalletResetBalance');
+jest.mock('../../../flows/WalletTransactions');
+jest.mock('../../../flows/WalletTransfer');
+jest.mock('../../../flows/WalletWithdrawal');
+jest.mock('../../../modules', () => ({
+    CashierLocked: jest.fn(({ children }) => <div>{children}</div>),
+    DepositLocked: jest.fn(({ children }) => <div>{children}</div>),
+    WithdrawalLocked: jest.fn(({ children }) => <div>{children}</div>),
+}));
+
+describe('WalletCashierContent', () => {
+    const renderComponent = (route: To) => {
+        const history = createMemoryHistory();
+        history.push(route);
+        render(
+            <Router history={history}>
+                <WalletCashierContent />
+            </Router>
+        );
+    };
+
+    beforeEach(() => {
+        (WalletDeposit as jest.Mock).mockReturnValue(<div>WalletDeposit</div>);
+        (WalletFiatOnRamp as jest.Mock).mockReturnValue(<div>WalletFiatOnRamp</div>);
+        (WalletResetBalance as jest.Mock).mockReturnValue(<div>WalletResetBalance</div>);
+        (WalletTransactions as jest.Mock).mockReturnValue(<div>WalletTransactions</div>);
+        (WalletTransfer as jest.Mock).mockReturnValue(<div>WalletTransfer</div>);
+        (WalletWithdrawal as jest.Mock).mockReturnValue(<div>WalletWithdrawal</div>);
+    });
```

Comment:
Hi @lubega-deriv, I don't think this is necessary, because the `returnValues` here are static, so they'll always be the same for each test. I think it's better to move the mocks to the global scope of the file. 

```suggestion
    beforeEach(() => {
        (WalletDeposit as jest.Mock).mockReturnValue(<div>WalletDeposit</div>);
        (WalletFiatOnRamp as jest.Mock).mockReturnValue(<div>WalletFiatOnRamp</div>);
        (WalletResetBalance as jest.Mock).mockReturnValue(<div>WalletResetBalance</div>);
        (WalletTransactions as jest.Mock).mockReturnValue(<div>WalletTransactions</div>);
        (WalletTransfer as jest.Mock).mockReturnValue(<div>WalletTransfer</div>);
        (WalletWithdrawal as jest.Mock).mockReturnValue(<div>WalletWithdrawal</div>);
    });
```

Reply:
Updated 👍